mslice/models/slice/matplotlib_slice_plotter.py

```diff
@@ -6,7 +6,11 @@
 class MatplotlibSlicePlotter(SlicePlotter):
     def __init__(self, slice_algorithm):
         self._slice_algorithm = slice_algorithm
-        self._colormaps = ['viridis', 'jet', 'summer', 'winter', 'coolwarm']
+        import matplotlib
```

Comment:
Can we move the import and version check to the module-level near the top of the file and use [`distutils.version.LooseVersion`](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/plugins/algorithms/StringToPng.py#L37) instead? If we need to have a check anywhere else in the file in the future it would be better to do it once and set a flag such as `MPL_COMPAT`?